### PR TITLE
[SCB-1144]change samples use log4j2

### PR DESCRIPTION
**Pull request by heyile**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: samples/auth-sample/auth-consumer/pom.xml [hunk 1]
```diff
@@ -29,6 +29,12 @@
     <dependency>
       <groupId>org.apache.servicecomb</groupId>
       <artifactId>transport-highway</artifactId>
+      <exclusions>
```
Comment (wujimin) [inline samples/auth-sample/auth-consumer/pom.xml]: is this necessary?

File: samples/auth-sample/auth-consumer/src/main/resources/log4j2.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!--this is sample configuration, please modify as your wish-->
+
+<configuration>
+  <Properties>
+    <Property name="log_path">./samples/auth-sample/auth-consumer/target/log/</Property>
```
Comment (wujimin) [inline samples/auth-sample/auth-consumer/src/main/resources/log4j2.xml]: current dir is not auth-consumer?

File: samples/auth-sample/auth-consumer/pom.xml (hunk 1 above)
Comment (heyile) [reply samples/auth-sample/auth-consumer/pom.xml]: of course. If do not  have this ...
the log is as bellow.
```
log4j:WARN No appenders could be found for logger (org.springframework.core.env.StandardEnvironment).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```

File: samples/auth-sample/auth-consumer/src/main/resources/log4j2.xml (hunk 2 above)
Comment (heyile) [reply samples/auth-sample/auth-consumer/src/main/resources/log4j2.xml]: should I change the log dir ?

**Comment by liubao68** [comment]
Why we need to do this change? Is it OK we add only one sample to show how to use log4j2?

**Pull request merged**